### PR TITLE
Removed a deprecated option

### PR DESCRIPTION
**Pull request by mgouiza**
Pull Request Checklist. Please read and check each box with an X. Delete any part not applicable. Ask on the [forum](https://community.geodynamics.org/c/aspect) if you need help with any step.

*Describe what you did in this PR and why you did it.*

### Before your first pull request:

* [x] I have read the guidelines in our [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md) document.

### For all pull requests:

* [x] I have followed the [instructions for indenting my code](../blob/main/CONTRIBUTING.md#making-aspect-better).

### For new features/models or changes of existing features:

* [ ] I have tested my new feature locally to ensure it is correct.
* [ ] I have [created a testcase](http://www.math.clemson.edu/~heister/manual.pdf#sec%3Awriting_tests) for the new feature/benchmark in the [tests/](../blob/main/tests/) directory.
* [ ] I have added a changelog entry in the [doc/modules/changes](../blob/main/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

**Review by MFraters — APPROVED** [review]

File: include/aspect/compat.h [hunk 1]
```diff
@@ -30,53 +30,7 @@
...
-                                   mpi_type_id(buffer),
-                                   root,
-                                   comm);
-        AssertThrowMPI(ierr);
-        total_sent_count += current_count;
-      }
-
-  }
-#endif
+using dealii::Utilities::MPI::broadcast;
```
Comment (bangerth) [inline include/aspect/compat.h:33]: This is correct, but not correctly indented. github is probably going to ask you to indent this as it was before, i.e., by two spaces.

**Review by bangerth — CHANGES_REQUESTED** [review]

File: include/aspect/compat.h (hunk 1 above)
Comment (mgouiza) [reply include/aspect/compat.h:33]: Sure, Thanks

**Review by bangerth — APPROVED** [review]
Yes, thank you!

**Review by bangerth — APPROVED** [review]
Yes, thank you!

**Pull request merged**